frontend/containers/DocumentEdit/DocumentEdit.jsx

```diff
@@ -158,9 +156,7 @@ class DocumentEdit extends Component {
       fields: fields.other
     }]
     const activeSection = this.props.section || sections[0].slug
-    const hasValidationErrors = Object.keys(document.validationErrors).filter(field => {
-      return document.validationErrors[field]
-    }).length
+    const hasValidationErrors = Object.keys(document.validationErrors).filter(field => document.validationErrors[field]).length
```

Comment:
Just trying to avoid the line being too long (it probably already was before you changed).